src/EVM/Types.hs

```diff
@@ -309,10 +309,11 @@ data Expr (a :: EType) where
   -- A restricted view of a contract that does not include extraneous metadata
   -- from the full constructor defined in the VM state
   C ::
-    { code    :: ContractCode
-    , storage :: Expr Storage
-    , balance :: Expr EWord
-    , nonce   :: Maybe W64
+    { code     :: ContractCode
+    , storage  :: Expr Storage
+    , tStorage :: Expr Storage
```

Comment:
can we call this `transientStorage`? I think it would be clearer

Reply:
We need to ask @d-xo , they are very much in favour of short names, AFAIK

Reply:
So d-xo has no strong feelings. I am in favour of tStorage, but up to you, push a change please in case you'd like this to be transientStorage -- although I really find that overly long. Maybe something smaller like trStorage?